scripts/reloadable_run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Run the different enclave servers as follows:
+# bash scripts/reloadable_run.sh --port 9081 --name "canada-domain"
+# bash scripts/reloadable_run.sh --port 9082 --name "italy-domain"
+# bash scripts/reloadable_run.sh --port 9083 --name "canada-enclave" --node_type "enclave"
+
+# And get the NodeHandlers as follows (note that the handlers point to the correct objects but are buggy in their attributes)
+# canada_domain = sy.orchestra.launch(port=9081, deploy_to="remote")
+# italy_domain = sy.orchestra.launch(port=9082, deploy_to="remote")
+# canada_enclave = sy.orchestra.launch(port=9083, deploy_to="remote")
+
+# Default values
+PORT=9694
+HOST="0.0.0.0"
+FACTORY="--factory"
+RELOAD="--reload"
```

Comment:
I tried this in jupyter and it seemed to cause a lot of weird behavior, where different code versions were being loaded at the same time, possibly because of the way jupyter manages threads of memory, where as when we do this in the container the entire process restarts so there is no mismatch between code versions.

I would love it if you have solved this! Has it been working for you?

Reply:
@madhavajay It worked for me but I'm making a few changes

- Will point it to `dev`, since it is more generally useful than Enclave stuff
- Will remove the script so that it works in python (so that all existing notebooks should work as is)

Will ping you when I have this ready.

Reply:
For some reason, unknown to mankind, this has moved [this PR](https://github.com/OpenMined/PySyft/pull/8971)